docker-compose-developer.yml

```diff
@@ -208,7 +208,7 @@ services:
     container_name: mysql
     labels: *oncall-labels
     image: mysql:8.0.32
-    command: --default-authentication-plugin=mysql_native_password --character-set-server=utf8mb4 --collation-server=utf8mb4_unicode_ci
+    command: --default-authentication-plugin=mysql_native_password --character-set-server=utf8mb4 --collation-server=utf8mb4_unicode_ci --max_connections=1024
```

Comment:
Increasing `max_connections` for local concurrency tests that open more connections than default number (151)

Reply:
what do you think about adding a `./dev/mysql.ini` file?

We have four containers (two in both this file + `docker-compose-mysql-rabbitmq.yml`) which run the `mysql` image. We could de-dupe this config and just volume mount this new file into all of these containers.

Reply:
All 4 mysql containers share only these two settings: `--character-set-server=utf8mb4` and  `--collation-server=utf8mb4_unicode_ci`. Not sure if deduping those is worth of the extra complexity added by volume mounting the `ini` file. I'll take a closer look at this, I think it should be outside of the scope of this PR.